docs/developer-docs/latest/plugins/i18n.md

```diff
@@ -338,17 +338,17 @@ This request:
 
 ### Updating an entry
 
-To update an existing localized entry, send a `POST` request to the appropriate URL depending on the type of content:
+To update an existing localized entry, send a `PUT` request to the appropriate URL depending on the type of content:
 
 | Content-Type    | Request URL format                           |
 | --------------- | -------------------------------------------- |
-| Collection type | `POST /api/{content-type}/:id/localizations` |
-| Single type     | `POST /api/{content-type}/localizations`     |
+| Collection type | `PUT /api/{localized-content-type}/:id/?field={newValue}` |
+| Single type     | `PUT /api/{localized-content-type}/?field={newValue}`     |
```

Comment:
Here I think having something more descriptive of the required params: `PUT /api/{localized-content-type}/?locale={locale}`